tests/controls/gridtest.cpp

```diff
@@ -806,4 +812,92 @@ void GridTestCase::WindowAsEditorControl()
 #endif
 }
 
+void GridTestCase::ResizeScrolledHeader()
+{
+    // TODO the test is works only under Windows unfortunately
+#if wxUSE_UIACTIONSIMULATOR && defined(__WXMSW__)
+    int const startwidth = m_grid->GetColSize(0);
+    int const draglength = 100;
+
+    m_grid->AppendCols(8);
+    m_grid->Scroll(5, 0);
+    m_grid->Refresh();
+    m_grid->Update();
+
+    wxRect rect = m_grid->CellToRect(0, 1);
+    wxPoint point = m_grid->CalcScrolledPosition(rect.GetPosition());
+    point = m_grid->ClientToScreen(point
+                                   + wxPoint(m_grid->GetRowLabelSize(),
+                                             m_grid->GetColLabelSize())
+                                   - wxPoint(0, 5));
+
+    wxUIActionSimulator sim;
+
+    wxYield();
+    sim.MouseMove(point);
+
+    wxYield();
+    sim.MouseDown();
+
+    wxYield();
+    sim.MouseMove(point + wxPoint(draglength, 0));
+
+    wxYield();
+    sim.MouseUp();
+
+    wxYield();
+
+    CPPUNIT_ASSERT_EQUAL(startwidth + draglength, m_grid->GetColSize(0));
+#endif
+}
+
+void GridTestCase::ColumnMinWidth()
+{
+    // TODO the test is works only under Windows unfortunately
+#if wxUSE_UIACTIONSIMULATOR && defined(__WXMSW__)
+    int const startminwidth = m_grid->GetColMinimalAcceptableWidth();
+    m_grid->SetColMinimalAcceptableWidth(startminwidth*2);
+    int const newminwidth = m_grid->GetColMinimalAcceptableWidth();
+    int const startwidth = m_grid->GetColSize(0);
+    int const newwidth = startwidth + 20;
+
+    CPPUNIT_ASSERT(m_grid->GetColMinimalAcceptableWidth() < startwidth);
+
+    wxRect rect = m_grid->CellToRect(0, 1);
+    wxPoint point = m_grid->CalcScrolledPosition(rect.GetPosition());
+    point = m_grid->ClientToScreen(point
+        + wxPoint(m_grid->GetRowLabelSize(),
```

Comment:
This is very minor, but indentation here is very strange, I don't understand why is it done like this.